[L2S-405] adapt docu for microservice production versioning (#2933)
SoftwareAG/c8y-docs: content/microservice-sdk/general-aspects-bundle/microservice-manifest.md

*   If a new ZIP file for a microservice is uploaded but the version is the same as the previous, for example, "1.1.0", then there is no guarantee that the Docker image for the microservice will be updated.
*   If the version is a snapshot, for example, "1.1.0-SNAPSHOT", then Docker will update the image on each ZIP upload.

> The snapshot postfix means that the image build is a snapshot of your application at a given time and it is still under development.

Reviewer: ```suggestion
The "-SNAPSHOT" postfix means that the image build is a snapshot of your application at a given time and it is still under development.
```

> When the microservice is ready for a production release, the "-SNAPSHOT" postfix must be removed, and a final version number (e.g., "2.1.0") should be used.

Reviewer: ```suggestion
When the microservice is ready for a production release, the "-SNAPSHOT" postfix must be removed, and a final version number (for example, "2.1.0") should be used.
```

> When the microservice is ready for a production release, the "-SNAPSHOT" postfix must be removed, and a final version number (for example, "2.1.0") should be used.

Reviewer 2: ```suggestion
When the microservice is ready for a production release, the "-SNAPSHOT" postfix must be removed, and the micorservice must be uploaded with a unique (higher) version number (for example, "2.1.0").
```

> In production, a new microservice is uploaded whenever a new version is specified different to the previously uploaded versions.

Reviewer 3: I'm not sure, but isn't any image uploaded and deployed, independent from the version ID, i. e. with identical version IDs? 
Any code change should get a new image ID

Author: In case of SNAPSHOT every uploaded microservice is installed. But in case of SNAPSHOT is missing only those microservices are uploaded and installed which have a newer version. If you upload a microservice of an old version this microservice is not installed.

> In production, a new microservice is uploaded whenever a new version is specified different to the previously uploaded versions.

Author: ```suggestion
In production, a new microservice is uploaded whenever a new version is specified different to the previously uploaded versions.
```
In production, a new microservice is installed whenever a new version is specified different to the previously installed.

> In production, a new microservice is uploaded whenever a new version is specified different to the previously uploaded versions.

Author: ```suggestion
In production, a new microservice is installed whenever a new version is specified different to the previously installed versions.
```

> In production, a new microservice is uploaded whenever a new version is specified different to the previously uploaded versions.

Reviewer 2: ```suggestion
```
